src/xdp/program.c

```diff
@@ -653,6 +654,7 @@ static const ebpf_program_data_t EbpfXdpProgramData = {
     .context_create = XdpCreateContext,
     .context_destroy = XdpDeleteContext,
     .required_irql = DISPATCH_LEVEL,
+    .capabilities = {.supports_context_header = TRUE},
```

Comment:
super nit (i wouldn't bother fixing): the general style is each field gets its own line of code, and is always followed by a `,`.

so this ideally would be:

```
.capabilities = {
    .supports_context_header = TRUE,
},
```